src/main/java/net/sf/jabref/gui/BasePanel.java

```diff
@@ -193,6 +193,8 @@
     // when a Back or Forward operation is being processed:
     private boolean backOrForwardInProgress;
 
+    private final Map<String, EntryEditor> entryEditors = new HashMap<>();
```

Comment:
If I see it correctly, then this map is always empty. Even if the content selectors need to be updated, I would strongly prefer that the event framework is used instead of push-forward through the BasePanel.